articles/nginx-reverse-proxy/index.md

```diff
@@ -0,0 +1,147 @@
+# Deploying Multiple Applications to VM with NGINX as a Reverse Proxy
+
+## Introduction
+Ever wondered how more than one application is being deployed to the same machine, and how traffic is being routed to the corresponding applications?
+Keep reading to know all about it!
+## Goals
+By the end of the article, one would be able to understand
+- What a Reverse Proxy is?
+- What NGINX is?
+- How does it help in managing multiple applications?
+- How to leverage NGINX as a Reverse Proxy?
+
+### Reverse Proxy
+A reverse proxy is a type of proxy server that retrieves resources on behalf of a client from one or more servers. These resources are then returned to the client, appearing as if they originated from the server itself.   
+
+### NGINX
+We will be using NGINX which would be serving the purpose of a Reverse Proxy. NGINX is a web server that can be used as a reverse proxy, load balancer, mail proxy and HTTP cache. 
```

Comment:
Add a bit of background about Nginx and suggest alternatives such as Varnish. You should link to an article discussing Nginx further.

Reply:
Thanks, I have added the details you've mentioned.